api/openapi.yaml

```diff
@@ -1237,7 +1200,13 @@ components:
         scanFamiliesConfig:
           $ref: '#/components/schemas/ScanFamiliesConfig'
         scope:
-          $ref: '#/components/schemas/ScanScopeType'
+          description: |
```

Comment:
We should change the examples of scanConfig.json wherever we have them in the project.